### ISSUE TITLE
about the downstream task

### ISSUE THREAD

**Issue by YanhaoWu** [PRBonn/segcontrast#13]
Hello!
I've noticed that you trained on KITTI for detection tasks and used PointRcnn. Can you share the relevant codes?
Thank you !

**Comment by nuneslu** [comment]
Hi, to run it I have created a docker image from OpenPCDet [repo](https://github.com/open-mmlab/OpenPCDet) and updated the models with our pre-trained model. You can pull the docker image [here](https://hub.docker.com/r/nuneslu/segcontrast_openpcdet) the config files are also on the docker image, please see the OpenPCDet repo for instruction about how to run it.

**Comment by YanhaoWu** [comment]
Thank you very much.
I have asked you a lot questions and you answered them! I am very grateful for that. 
Now I'm a big fan of yours！

**Comment by nuneslu** [comment]
No worries! I'm glad to see people interested and using our work. :)

**Comment by YanhaoWu** [comment]
> No worries! I'm glad to see people interested and using our work. :)

I have downloaded the docker image you provided. I hope to reproduce the experiment about detection in your paper. One of them is set to use the 5% label for experiment, but I can't find out which 5% label you use? Can you help me? I checked several yaml files under the cfg file. It seems that the settings of these files are the same?
![RV`KU9J S(CNF5CB%85Y67M](https://user-images.githubusercontent.com/79640383/176350172-1dd7d060-6d00-4cb7-9a41-2b23a70c7baf.png)

**Comment by nuneslu** [comment]
Hi, yes I've checked and actually the splits are not in the docker image, sorry for that. But I have used [those](https://github.com/zaiweizhang/OpenPCDet/tree/master/data/kitti/split_infos) from DepthContrast baseline (which also used the OpenPCDet repo), you can just download it and add to the directory `OpenPCDet/data/kitti/split_infos`

**Comment by nuneslu** [comment]
btw I have updated the docker image with the `split_infos` dir

**Comment by YanhaoWu** [comment]
> Hi, yes I've checked and actually the splits are not in the docker image, sorry for that. But I have used [those](https://github.com/zaiweizhang/OpenPCDet/tree/master/data/kitti/split_infos) from DepthContrast baseline (which also used the OpenPCDet repo), you can just download it and add to the directory `OpenPCDet/data/kitti/split_infos`
Thank you for you reply!

**Comment by YanhaoWu** [comment]
> Hi, to run it I have created a docker image from OpenPCDet [repo](https://github.com/open-mmlab/OpenPCDet) and updated the models with our pre-trained model. You can pull the docker image [here](https://hub.docker.com/r/nuneslu/segcontrast_openpcdet) the config files are also on the docker image, please see the OpenPCDet repo for instruction about how to run it.

Hi ! I found PointContrast and DepthContrast model parameters in this image. Did you get them through pre training on KITTI?

**Comment by nuneslu** [comment]
Hi, in the docker image `latest` tag the weights are the ones used in the paper for segcontrast and the baselines. I have recently created an image to run it also with cuda 11.1 in another tag and for this image, the pretrained weights are not the correct ones.

**Comment by YanhaoWu** [comment]
> Hi, in the docker image `latest` tag the weights are the ones used in the paper for segcontrast and the baselines. I have recently created an image to run it also with cuda 11.1 in another tag and for this image, the pretrained weights are not the correct ones.

Thank you!
